pkg/web_app/lib/src/widget/downloads_chart/widget.dart

```diff
@@ -47,6 +51,34 @@ void create(HTMLElement element, Map<String, String> options) {
     weeksToDisplay,
   );
 
+  final minorDisplayLists = prepareWeekLists(
+    data.totalWeeklyDownloads,
+    data.minorRangeWeeklyDownloads,
+    weeksToDisplay,
+  );
+
+  final patchDisplayLists = prepareWeekLists(
+    data.totalWeeklyDownloads,
+    data.patchRangeWeeklyDownloads,
+    weeksToDisplay,
+  );
+
+  final versionModesLists = {
+    'Major': majorDisplayLists,
+    'Minor': minorDisplayLists,
+    'Patch': patchDisplayLists
+  };
+
+  final versionModes = document.getElementsByName(versionsRadio);
+  for (int i = 0; i < versionModes.length; i++) {
```

Comment:
Please make you life easier with `toList()` extension from:
https://github.com/dart-lang/pub-dev/blob/9e7f4b958d98195229c2302ac906e32915bd2bf4/pkg/web_app/lib/src/web_util.dart#L9-L31

Reply:
done